### PR TITLE
Enhance: Adding issue templates

### PR DESCRIPTION
**Pull request by shadmansaleh**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: Feature Resuest
+about: Request a feature for lualine
+title: 'Feat: '
+labels: mew feature
```
Comment (hoob3rt) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]: mew? haha

otherwise lgtm

Comment (shadmansaleh) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Ha ha . Fixed :D

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+labels: bug
+
+---
+
+<!-- Before reporting: search existing issues and ensure you are running the latest nightly of neovim and the latest version of lualine.-->
+
+### Self Checks
+
+<!-- Check these boxes by placeing an X in the middle -->
+- [ ] I'm useing latest lualine and neovim nightly.
```
Comment (hoob3rt) [inline .github/ISSUE_TEMPLATE/bug_report.md:14]: using

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+- [ ] I Didn't find the issue in exsisting issues or prs.
```
Comment (hoob3rt) [inline .github/ISSUE_TEMPLATE/bug_report.md:15]: didn't should be lowercase

**Review by hoob3rt — APPROVED** [review]
fix these two types and then try your new privileges and merge this one :smile:

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (hoob3rt) [reply .github/ISSUE_TEMPLATE/bug_report.md]: should be `using the latest` actually

Comment (shadmansaleh) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Fixed . Grammer was naver my thing 😅

Comment (hoob3rt) [reply .github/ISSUE_TEMPLATE/bug_report.md]: np, ty 👍

**Comment by shadmansaleh** [comment]
merged 😃

**Comment by hoob3rt** [comment]
> merged 😃

great! Thanks 👍

**Pull request merged**